configurations/manager/rocky8.yaml

```diff
@@ -0,0 +1,3 @@
+ami_id_monitor: 'ami-011ef2017d41cb239'  # Rocky Linux 8 (Official) image
```

Comment:
I suppose we should use the approach with resolve:ssm parameter introduced recently.
I'm not sure I have permissions to create such parameter:
```
$ aws ssm put-parameter --name "/aws/service/rocky/release/8/latest/x86_64" --value "ami-011ef2017d41cb239" --type "String"

An error occurred (AccessDeniedException) when calling the PutParameter operation: No access to "/aws/" namespace: aws/service/rocky for the account: 797456418907
```

@fruch Could you please help with that?

Reply:
The idea with those is to read the official image of a disto (if it has ones)

Maybe we can put values into SSM, but we never intended to do so.

In theory every thing in the marketplace should be searchable via its product id, i.e. as I'm doing for Ubuntu, but I haven't found a clear way of obtaining it from the marketplace (didn't have time to dig deeper)

Reply:
So, I've tried to search for it as well - didn't find anything for Rocky describing public parameters (https://docs.aws.amazon.com/systems-manager/latest/userguide/parameter-store-finding-public-parameters.html).

Therefore, I'm keeping the ami_id in the configuration file.

Reply:
Better to specify region in comment too.